i18n/ja.toml

```diff
@@ -77,3 +77,13 @@ other = "If you want to get more involved by contributing to {{ .Site.Title }},
 other = "You can find out how to contribute to {{ .Site.Title }} in our"
```

Comment:
Just realized that there are some more untranslated lines/strings in this file:

https://github.com/google/docsy/blob/fa81a86be8bf130d94a43b8898c6716819fdae87/i18n/ja.toml#L63C1-L79C34

@rinsuki: are you able to translate these lines as well? Your help is appreciated!

Reply:
technically I can directly translate word-by-word, but I think it wouldn't be a good translate (If I do that, japanese speakers (like me) feels it is translated from other language rather than written by native), but I don't have a idea to good paraphrase for its text right now.

by the way, `community_how_to` assume text ends with `community_guideline`:

https://github.com/google/docsy/blob/fa81a86be8bf130d94a43b8898c6716819fdae87/layouts/partials/community_links.html#L18-L21

but some languages wants to have additional text *after* link, it would be good to have something like `community_how_to_suffix`.

Reply:
Ok, thanks @rinsuki. I'll merge this as an increment, and we can address the other fields as necessary. Thanks for pointing that out @deining.